tests/integration/replay/replay_tests.bash

```diff
@@ -35,6 +35,7 @@ fi
 #
 diff replay.out $replay_tests_dir/test_output.good
 if [ $? -ne 0 ]; then
+    /bin/rm -f replay.out umpire*replay umpire*log
```

Comment:
I ran into a problem where tests were failing if multiple replay files existed when running the test - they never got cleaned up because the test failed.

Reply:
Thanks!